<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>milan338</author>
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="bottom_bar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>1001</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(27, 27, 35);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QPushButton" name="btn_serial_clear">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>440</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(27, 27, 35);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(74, 74, 74);
}

QPushButton:pressed  {
	background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string>Clear Output</string>
   </property>
  </widget>
  <widget class="QWidget" name="top_bar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(27, 27, 35);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QLineEdit" name="input_serial_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>681</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 47, 59);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_serial_send">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(27, 27, 35);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(74, 74, 74);
}

QPushButton:pressed  {
	background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="text_scroll_region">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>1001</width>
     <height>361</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 47, 59);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="text_scroll_region_widgets">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>984</width>
      <height>361</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="style" stdset="0">
     <string>scrollregion</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="text_display">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_autoscroll">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(27, 27, 35);</string>
   </property>
   <property name="text">
    <string>Auto-Scroll</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="switch_placeholder"/>
  </widget>
  <widget class="QPushButton" name="btn_input_type">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(27, 27, 35);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(74, 74, 74);
}

QPushButton:pressed  {
	background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string>Input Type</string>
   </property>
  </widget>
  <zorder>text_scroll_region</zorder>
  <zorder>bottom_bar</zorder>
  <zorder>top_bar</zorder>
  <zorder>input_serial_text</zorder>
  <zorder>btn_serial_send</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>btn_serial_clear</zorder>
  <zorder>label_autoscroll</zorder>
  <zorder>btn_input_type</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
